webservice/signing/jar/src/main/java/org/eclipse/cbi/webservice/signing/jar/JarSigner.java

```diff
@@ -255,14 +116,23 @@ public Path signJar(Path jar, SignatureAlgorithm sigAlg, MessageDigestAlgorithm
 	 *         {@link ProcessBuilder} for format).
 	 */
 	private ImmutableList<String> createCommand(Path jar, SignatureAlgorithm sigAlg, MessageDigestAlgorithm digestAlg, String sigFile) {
-		ImmutableList.Builder<String> command = ImmutableList.<String>builder().add(jarSigner().toString());
-		
-		if (!Strings.isNullOrEmpty(httpProxyHost())) {
-			command.add("-J-Dhttp.proxyHost=" + httpProxyHost()).add("-J-Dhttp.proxyPort=" + httpProxyPort());
+		ImmutableList.Builder<String> command = ImmutableList.<String>builder().add(configuration().getJarSigner().toString());
+
+		if (!Strings.isNullOrEmpty(configuration().getJavaArgs())) {
+			String[] args = configuration().getJavaArgs().split(" ");
```

Comment:
I'd use [Guava's splitter](https://github.com/google/guava/wiki/StringsExplained#splitter), a more robust version of String#split, with less deceitful / more predictable results in many edge cases.